release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -245,6 +245,11 @@ With this update, you can provision IPv6 address spoke clusters from dual-stack
 
 With this release, you can pre-cache your application workload images before upgrading your applications on {sno} clusters with {cgu-operator-full}. For more information, see xref:../scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc#talm-prechache-user-specified-images-concept_ztp-talm[Pre-caching user-specified images with TALM on single-node OpenShift clusters].
 
+[id="ocp-4-14-ztp-support-custom-node-labels"]
+==== Support for adding custom node labels in the SiteConfig CR through GitOps ZTP
+
+With this update, you can create custom node labels in the `SiteConfig` CR to create custom roles for nodes in managed clusters. For more information about how to add custom labels, see xref:../scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc#ztp-deploying-a-site_ztp-deploying-far-edge-sites[Deploying a managed cluster with SiteConfig and GitOps ZTP] or xref:../scalability_and_performance/ztp_far_edge/ztp-manual-install.adoc#ztp-generating-install-and-config-crs-manually_ztp-manual-install[Generating GitOps ZTP installation and configuration CRs manually].
```

Comment:
From my view, the statement should be as below
With this update, you can add "nodeLabels" attribute in the SiteConfig CR to create custom roles and labels for nodes in managed clusters.

Reply:
Looks good to me other than above mentioned minor comment.